src/components/shared/Discord.jsx

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable max-len */
+import React from 'react';
+
+const Discord = () => {
+  React.useEffect(() => {
+    const script = document.createElement('script');
+    script.src = 'https://discord.com/invite/yQZ368XPrB';
+    script.async = true;
+    script.defer = true;
+    document.body.appendChild(script);
+    return () => {
+      document.body.removeChild(script);
+    };
+  }, []);
+
+  return (
+    <div
+      style={{
+        display: 'flex',
+        justifyContent: 'center',
+        alignContent: 'center',
+      }}
+    >
+      <div
+        className='apply-button'
```

Comment:
@babitarit at least copy properly. 
Discord button is completely different. Check the code of last year's website.